dir.traversal.targets

```diff
@@ -60,6 +60,11 @@
     <!-- To Serialize we use msbuild's batching functionality '%' to force it to batch all similar projects with the same identity
          however since the project names are unique it will essentially force each to run in its own batch -->
 
+    <!-- Coverage builds require serial execution. -->
+    <PropertyGroup>
+      <SerializeProjects Condition="'$(Coverage)' == 'true'">true</SerializeProjects>
```

Comment:
I'm actually having trouble figuring out why it isn't working. It seems like SerializeProjects should be getting set to `true` in CodeCoverage.targets. src/tests.builds wasn't previously importing dir.targets (and therefore it wasn't importing CodeCoverage.targets), but even after I add that import it isn't serializing the build. Another thing is that the code coverage tests don't need to be BUILT serially, just executed serially.